samples/MvcSample.Web/wwwroot/Scripts/Mvc.js

```diff
@@ -29,7 +31,7 @@ var MVC = (function () {
             return;
         }
 
-        if (Object.prototype.toString.call(value) === '[object Array]') {
+        if (Object.prototype.toString.call(value) === "[object Array]") {
```

Comment:
It doesn't change functionality, but it does enforce consistency. I suppose it depends on what the project's coding standards are. In my mind, consistency leads to a lower barrier to entry for contributors on this project.